Tasks/words.js

```diff
@@ -1,41 +1,18 @@
 // Count words in a string
+'use strict';
 
-Words = function (s) {
-  number_of_words_in_s = 0;
-  flag = false;
-  for (c of s) {
-    if (!flag) {
-      if (c === ' ') {
-        if (flag === true) {
-          flag = false;
-        } else {
-          flag = false;
-        }
-      } else {
-        if (flag === true) {
-          flag = true;
-        } else {
-          flag = true;
-        }
-        number_of_words_in_s++;
-      }
-    } else {
-      if (c === ' ') {
-        if (flag === true) {
-          flag = false;
-        } else {
-          flag = false;
-        }
-      } else {
-        if (flag === true) {
-          flag = true;
-        } else {
-          flag = true;
-        }
-      }
+const words = (s) => {
+  let numberOfWords = 0;
+  let inWord = false;
+  for (let i = 0; i < s.length; i++) {
+    if (s[i] === ' ') {
+        inWord = false;
```

Comment:
Can be implemented with: `inWord = !inWord;`

Reply:
Used ESlint and fixed in my repository: https://github.com/RostislavHundarev/fixed-exams